<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flowMonitor</class>
 <widget class="QMainWindow" name="flowMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>flowMonitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="4">
     <widget class="vtkQtChartWidget" name="realTimeChartWidget" native="true"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="vtkQtChartWidget" name="medTimeChartWidget" native="true"/>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="vtkQtChartWidget" name="longTimeChartWidget" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QTimeEdit" name="ctTimeDisplay">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>h:mm:ss AP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="buttonSetTime">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="infoLabel">
      <property name="text">
       <string>synchronize Time!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCalibrate"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Info about this Application</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>&amp;Calibrate</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save Log</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vtkQtChartWidget</class>
   <extends></extends>
   <header>vtkQtChartWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
